sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -536,7 +536,13 @@ void gnrc_rpl_recv_DIO(gnrc_rpl_dio_t *dio, ipv6_addr_t *src, uint16_t len)
 
         inst->mop = (dio->g_mop_prf >> GNRC_RPL_MOP_SHIFT) & GNRC_RPL_SHIFTED_MOP_MASK;
         inst->of = gnrc_rpl_get_of_for_ocp(GNRC_RPL_DEFAULT_OCP);
-        gnrc_rpl_dodag_init(inst, &dio->dodag_id);
+
+        if (iface == KERNEL_PID_UNDEF) {
+            iface = gnrc_ipv6_netif_find_by_addr(NULL, &ipv6_addr_all_rpl_nodes);
```

Comment:
@BytesGalore I amended a quick change in this line. I apparently did an error while rebasing, so it was using the static initializer for the all_rpl_nodes address. I changed that to use the global var instead as introduced by a former PR